CHANGELOG.md

```diff
@@ -89,6 +89,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 * Stop adding header search paths that do not contain any headers.  
   [Samuel Giddins](https://github.com/segiddins)
 
+* Fix returning absolute paths from glob, fixes issue with static framework and public headers.
+  [Morgan McKenzie](https://github.com/rmtmckenzie)
+  [#7463](https://github.com/CocoaPods/CocoaPods/issues/7463)
```

Comment:
nit: press enter

Reply:
Will do (saw that too but not until after I committed). I'll put that in with the unit test when I get a chance.